packages/modules/atlas-core/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   We added Language Selector widget to Atlas_Default and Atlas_TopBar responsive layouts.
 -   Minimal required version of Studio Pro is v9.12.0 and above.
+-   We added error style to datepicker dropdown and referenceSelector widgets.
```

Comment:
@Ahmad-Elsayed Shouldn't be comma separated? 🤔

Reply:
done